### PR TITLE
Use inline training data in generated Console Project file.

### PR DESCRIPTION
**Pull request by LittleLittleCloud**
### Code Sample

``` c#
// This file was auto-generated by ML.NET Model Builder. 

using System;
using SampleRecommendation.Model;

namespace SampleRecommendation.ConsoleApp
{
    class Program
    {
        static void Main(string[] args)
        {
            // Create single instance of sample data from first line of dataset for model input
            ModelInput sampleData = new ModelInput()
            {
                UserId = 1,
                MovieId = 1,
                Timestamp = 9.649827E+08F,
            };

            // Make a single prediction on the sample data and print results
            var predictionResult = ConsumeModel.Predict(sampleData);

            Console.WriteLine("Using model to make single prediction -- Comparing actual Rating with predicted Rating from sample data...\n\n");
            Console.WriteLine($"userId: {sampleData.UserId}");
            Console.WriteLine($"movieId: {sampleData.MovieId}");
            Console.WriteLine($"timestamp: {sampleData.Timestamp}");
            Console.WriteLine($"\n\nActual Rating: {sampleData.Rating} \nPredicted Rating: {predictionResult.Score}\n\n");
            Console.WriteLine("=============== End of process, hit any key to finish ===============");
            Console.ReadKey();
        }
    }
}
```

#### Fix issue
- [Simply CodeGen](https://app.zenhub.com/workspaces/mlnet-tools-5cde1c97e3106e39e8ae08fc/issues/dotnet/machinelearning-modelbuilder/558) ( Item 1 )

### CODE REVIEW

File: src/Microsoft.ML.AutoML/TrainerExtensions/RecommendationTrainerExtensions.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ internal class MatrixFactorizationExtension : ITrainerExtension
             options.LabelColumnName = columnInfo.LabelColumnName;
             options.MatrixColumnIndexColumnName = columnInfo.UserIdColumnName;
             options.MatrixRowIndexColumnName = columnInfo.ItemIdColumnName;
+            options.Quiet = true;
```
Comment (LittleLittleCloud) [inline src/Microsoft.ML.AutoML/TrainerExtensions/RecommendationTrainerExtensions.cs]: This is for quite the verbose output from MatrixFactorization trainer

File: test/Microsoft.ML.CodeGenerator.Tests/UtilTest.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            var data = new[]
+            {
+                new
+                {
+                    Label = "label1",
+                    STR = "feature1",
+                    INT = 2,
+                    DOUBLE = 1.2,
+                    FLOAT = 1.223E+10F,
+                    TrickySTR = "ab\"\';@#$%^&-++==",
```
Comment (justinormont) [inline test/Microsoft.ML.CodeGenerator.Tests/UtilTest.cs]: Some others to test:
* `Single.NaN`
* `Single.PositiveInfinity`
* "" (empty string)
* 1 (as boolean)
* "T" (as boolean)

Comment (LittleLittleCloud) [reply test/Microsoft.ML.CodeGenerator.Tests/UtilTest.cs]: ### Resolved

Comment (justinormont) [reply test/Microsoft.ML.CodeGenerator.Tests/UtilTest.cs]: Thanks!

**Pull request merged**